core/controllers/classifier_test.py

```diff
@@ -265,8 +266,9 @@ def setUp(self):
         self.job_id = classifier_models.ClassifierTrainingJobModel.create(
             self.algorithm_id, interaction_id, self.exp_id, 1,
             datetime.datetime.utcnow(), self.training_data, 'Home',
-            feconf.TRAINING_JOB_STATUS_NEW, None, 1
+            feconf.TRAINING_JOB_STATUS_NEW, 1
```

Comment:
This change is going to require some operations to delete ClassifierTrainingJobModel on the test server, so we need to arrange for that before this PR is checked in.

Are there any other storage models which depend on ClassifierTrainingJobModel and need to be deleted/modified as well?

Reply:
Sure, will do.